invenio_app_ils/ui/backoffice/src/pages/ItemsSearch/__tests__/SearchBar.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
I might be terribly wrong, but these look like tests for the SearchBar input. I am under the impression that react-search-kit comes with one out of the box. The point I am trying to convey here is that it should not part of the ils application to test each library's functionality, if that happens to be the case.

Reply:
Well, I am overriding template and behaviour. React-SearchKit will come with tests for implemented behaviours, but if you override...